guides/file-uploads.md

```diff
@@ -0,0 +1,22 @@
+```
+query Node {
+  node(id: "msg_4023561927") {
+    ... on EmailMessage {
+      attachments {
+        nodes {
+          contentId
```

Comment:
What is contentId?

Reply:
field :contentId,
        String,
        null: true,
        description: 'The RFC2387 identifier (if from an email)'